cli/cmd/docs.go

```diff
@@ -64,8 +64,13 @@ func init() {
 }
 
 func GenerateMarkdownDocs(location string) error {
-	// if the location doesn't exist, we will create it for the user
-	if err := os.MkdirAll(location, 0755); err != nil {
+	// remove location before generating to ensure deleted commands are removed
+	err := os.RemoveAll(location)
```

Comment:
ughh.. This change actually deleted our code generator (`main.go`) 😭 

https://github.com/lacework/go-sdk/pull/1181/files#diff-5106360d170ed88b364e8f441ad401fe467da63c98ff6e936590812039dbf7c7

![Screen Shot 2023-04-06 at 4 21 52 PM](https://user-images.githubusercontent.com/5712253/230511035-dfdc0bcd-dc3f-4ea9-80c5-a8e6dc1b4f62.png)


GenerateMarkdownDocs